content/references/xrp-ledger-toml.md

```diff
@@ -325,6 +329,13 @@ NmluOEhBU1FLUHVnQkQ2N2tNYVJGR3ZtcEFUSGxHS0pkdkRGbFdQWXk1QXFEZWRGdj
 VUSmEydzBpMjFlcTNNWXl3TFZKWm5GT3I3QzBrdzJBaVR6U0NqSXpkaXRROD0ifQ==     
 ```
 
+Update your `xrp-ledger.toml` file as described in the sample output. See [here](xrp-ledger-toml.html#contents) for an example.
```

Comment:
Three things on this line:

1. We should avoid using "here" as link text; it doesn't provide much context about where the link goes. 
2. I think it would be helpful to be more specific about what it means to update "as described in the sample output".
3. This same file is served as `xrp-ledger-toml.html` so we can shorten the link href to just `#contents`.

Taking those into account, here a suggestion of how I'd word it:

```md
Update [the contents of your `xrp-ledger.toml` file](#contents) with the `attestation` block from the sample output.
```

Also, the sample output suggests to me that you need to update the `[validator_token]` in your `rippled` config file. If that's true, we should mention it here; if you can skip that step, we should also mention that here.

Reply:
Ah that makes sense. Thanks, just pushed up the new changes.